sdk/servicebus/service-bus/src/receivers/sessionReceiver.ts

```diff
@@ -478,6 +483,11 @@ export class SessionReceiverImpl<ReceivedMessageT extends ReceivedMessage | Rece
       },
       options
     );
+
+    return {
+      // TODO: coming in a future PR.
```

Comment:
Just curious if you saw something that would result in the approach for sessions to be different from the non sessions scenario.

Reply:
The approach might be the same but I wanted to think about it separately since it's a separate code path (blech).